AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -23,6 +23,8 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 yahoo.co.jp##.sn-modAd
+lazada.co.th###J_Smart_Banner_Bottom
```

Comment:
`lazada.co.th###J_Smart_Banner_Bottom`
Blank space. Can be fixed by removing `with-smart-banner` class, or otherwise.

![image](https://user-images.githubusercontent.com/8361299/165313576-d09efb99-eb2e-44e9-adc5-768e778c5884.png)